src/test/java/org/opensrp/repository/postgres/ClientsRepositoryTest.java

```diff
@@ -503,7 +503,7 @@ public void testNotInOpenMRSByServerVersion() {
 	@Test
 	public void shouldFindMembersByRelationshipId() {
 		List<Client> expectedClient = clientsRepository.findMembersByRelationshipId("0154839f-8766-4eda-b729-89067c7a8c5d");
-		assertEquals(expectedClient.size(), 2);
+		assertNotNull(expectedClient);
```

Comment:
I think in addition we need to return the count check here so that we are sure the test is accurate and not broken in future